test/e2e/virtualoutbound/virtualoutbound_k8s.go

```diff
@@ -13,6 +13,7 @@ import (
 
 func VirtualOutboundOnK8s() {
 	var k8sCluster Cluster
+	namespace := "virtual-outbound"
```

Comment:
Any idea why changing namespace helps here? Feels like this tests runs its own CP anyway.
Btw we could also move this test to universal and use test/e2e_env (I think we already do this for quite a few tests.

Reply:
Deletion of the namespace takes quite a lot of time. I wanted to avoid conflicts that's why I've changed `kuma-test` which we use in most of the tests. About the isolation problem: our tests are using helm to install control plane and it seems that sometimes removing `kuma-system` namespaces from the cluster takes too much time and so far I suspect that is the cause of fails  As you can see we are reusing the same K8s cluster and I think that is the case.